config/static.c

```diff
@@ -49,7 +49,7 @@ l_list_lcores(lua_State *l)
 	lua_newtable(l);	/* Result. */
 
 	/* Only list slave lcores because the master lcore is special. */
-	RTE_LCORE_FOREACH(i) {
+	RTE_LCORE_FOREACH_SLAVE(i) {
```

Comment:
Instead of just reverting back to `RTE_LCORE_FOREACH()`, add another patch to this pull request to drop the comment above AND to correct the comment of the function.

Reply:
The idea is to let the master lcore be used by other block as already happens, and add code to the LLS block to deal with the log.